abi/src/test/java/org/web3j/abi/TypeDecoderTest.java

```diff
@@ -348,7 +348,7 @@ public void testStaticArray() throws Exception {
         StaticArray2 arr =
                 (StaticArray2)
                         TypeDecoder.instantiateType("uint256[2]", new long[] {10, Long.MAX_VALUE});
-        assert (arr instanceof StaticArray2);
+        assert (arr != null);
 
```

Comment:
Are you sure about these, you probably want something like
`assertThat(arr, instanceOf(StaticArray2.class));`